### ISSUE TITLE
Request: Counter for Truce Tower

### ISSUE THREAD

**Issue by franzalex** [dsiekiera/foe-helfer-extension#1444]
In version 2.5.0.0, a box was added that showed the number of Kraken charges that were left. This greatly enhanced preparations for battles. I would like request that something similar to be done for the Truce Tower (formerly Dynamic Tower) so that we can see how many awards are left.

Thank you.

--- 

**PS:** If this is given consideration, then I think something similar could be done for the _Space Carrier_ and other similar GBs.

**Comment by andrgin** [comment]
Space carrier and Himeji are already shown.

For Truce tower: We will see, but I dont not have one, so it is hard to test for me.

**Comment by franzalex** [comment]
I didn't know you needed to have that GB in order to test for it.

Will be faster/easier if I send you the HAR (entries from Network tab in Developer tools) or Fiddler log while motivating other players?

**Comment by franzalex** [comment]
@andrgin  
I have attached a filtered HAR file of me aiding friends and guild mates with Truce Tower active.

When the ```response.content.text``` is unescaped, among the results obtained is the following:

```js
[
    /* one or more items unrelated to this request */
    {
        "responseData": [
            {"entityId":6,"amount":10,"value":160,"type":"supplies_boost","__class__":"LimitedBonus"},
            {"entityId":238,"amount":88,"value":220,"type":"money_boost","__class__":"LimitedBonus"},
            {"entityId":256,"amount":45,"value":215,"type":"supplies_boost","__class__":"LimitedBonus"},
            {"entityId":326,"amount":29,"value":1,"type":"aid_goods","__class__":"LimitedBonus"},
            {"entityId":5165,"amount":13,"value":41.74,"type":"first_strike","__class__":"LimitedBonus"},
            {"entityId":8774,"amount":-1,"value":31,"type":"contribution_boost","__class__":"LimitedBonus"}
        ],
        "requestClass": "BonusService",
        "requestMethod": "getLimitedBonuses",
        "requestId": 44,
        "__class__": "ServerResponse"
    }
]
```

The item with type **"aid_goods"** corresponds to the counter value for the Truce Tower.  

@andrgin Is the information I have provided enough to include a counter for the Truce Tower or will your team need more details?  
Thank you


-----

**Attachments:**

 * [**```en2.forgeofempires.com - truce tower-3.filtered-1.zip```**](https://github.com/dsiekiera/foe-helfer-extension/files/4903010/en2.forgeofempires.com.-.truce.tower-3.filtered-1.zip)

**Comment by dsiekiera** [comment]
That doesn't make sense. It appears in the guild battles, the GEX, at the neighbor's house, etc.

And there you can't really pimp a friend or a guild member.

**Comment by franzalex** [comment]
@dsiekiera 
Yes, I now this isn't of any use in the battlegrounds and expedition.

My request is for it to be shown when you visit another player's city. If the counter appears there then you'll know how many more goods can be obtained from aids.
